tools/hashes/eksctl

```diff
@@ -1,5 +1,5 @@
-1cf804f0a135ce20e038703ab234eb3a552b7b636809613e13c7215cf55799c67e553b45397a86d6e713c833c3fbc2f5069d5d6f84bd1f5c5cd64efe94cd8bf7  eksctl_0.112.0.tar.gz
-45f593b9d594c3d1151cad7ff03faff9f36843e768177502f36f1b9622c98ee9e83f752d3c7799c4248fb8dac922c1dc23aa8f11177e127c83e8442a7eac3d44  eksctl_0.112.0_linux_amd64.tar.gz
-2994cae53100ded6a9f50bbc43168b6f83f9a926b66d98c0e5315da4350f30d7822ccaeb242733f28346ba07668ad3bd453e7d91fd926aa46023d471ca9cd394  eksctl_0.112.0_linux_arm64.tar.gz
-17446c1c529e40948a388b3ae75c84dabe19ef2e3c446a95a20062039e01182d313df99b1203fff4cc944ad6dccea9df0fcf6b33fe8998ae7217b7a939638dea  eksctl_anywhere_0.12.1.tar.gz
-f59fc52ab809abc0687ec3743e4880207ccbba3881fb30b7fc9e8f3380e5fc815b5439357d592d902f6dd4d8088c3094ca0c271b8762222ad51d81baa1a062dd  eksctl_anywhere_0.12.1_23_linux_amd64.tar.gz
+53031a48bf3545a44cde2b0a8dce11d30bb465c107a4e672142e53878d02caec0a18582f0e7aa01408f4a1d282a6742586d87c42856d2bb5e2d759c9495bc86d  eksctl_0.120.0.tar.gz
+170f5a37611c3bc7c99650751df8a30cfb9514d854b45aabb6b33842ec4497fddec415f80a4af78a995577cdc71742dc6cf5c6c25bab2eb0c17cbd6a254408fd  eksctl_0.120.0_linux_amd64.tar.gz
+fea2381e3ed230462cf677adc603da0c54c5186325f24c2d36c90e8f49daf0f22d976e5c9c8b7e0a775fb5e0370763b817a8d1b5bb152964e21625d3bd4bc21f  eksctl_0.120.0_linux_arm64.tar.gz
+75a007999c45c405013bea56900a5b1facd08c4d3e40b60251fd57106dd28d19ac9ae1c39c269f25d7c7d4c41e805454b5169c5da71d1baf3eca5e6f038d90a2  eksctl_anywhere_0.12.2.tar.gz
+527ebc61d18d6d46e511219c215f4746a03cfe2fffa2447ed1344beec0baffcc7f2fa828bc0a414e877b1bcc13710b7de315ee6eb6c277065a3553a69d452dcc  eksctl_anywhere_0.12.2_24_linux_amd64.tar.gz
```

Comment:
Does eksctl anywhere not have arm binaries?

Reply:
No they do not provide ARM binaries. I added a note here:
https://github.com/bottlerocket-os/bottlerocket-test-system/blob/123e1d18a79c8d81f20134e35dbee8311b9768b4/tools/Dockerfile#L56-L57